.devcontainer/scripts/ensure-dapr.sh

```diff
@@ -30,8 +30,8 @@ init_dapr()
 
 ROOT_DIRECTORY=$( realpath "$( cd -- "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )/../.." )
 echo $ROOT_DIRECTORY
-DEFAULT_DAPR_CLI_VERSION=$(cat $ROOT_DIRECTORY/app/AppManifest.json | jq .[].dependencies.dapr.cli.version | tr -d '"')
-DEFAULT_DAPR_RUNTIME_VERSION=$(cat $ROOT_DIRECTORY/app/AppManifest.json | jq .[].dependencies.dapr.runtime.version | tr -d '"')
```

Comment:
please leave the appManifest for now since we might need a new concept at all. At least we should sync this in all repos.